### ISSUE TITLE
wave_spec_file input setting in ice_in

### ISSUE THREAD

**Issue by apcraig** [CICE-Consortium/CICE#415]
In CICE, the wave_spec_file is defined internally in ice_forcing.F90 as

```
  wave_spec_dir = ocn_data_dir
  spec_file = trim(wave_spec_dir)//'/'//trim(wave_spec_file)
```
I think the wave_spec_file variable in ice_in should include the full path with the inputdata path automatically added.   Leveraging the ocn_data_dir is this case is probably not a great idea.  It also prevents us from moving the file to a different location easily or having a user point to a new file in their local space easily.  We will need to modify this before we can move the ww3 fsd file in our input data area.

**Comment by lettie-roach** [comment]
I agree, I have been having to modify the code to point to the right file. I can set spec_file = trim(wave_spec_file). To get the inputdata path added, is this a matter of changing all the CICE/configuration/scripts/options/set_nml.* files?

**Comment by apcraig** [comment]
There is really only one file in set_nml that needs to change, that's the fsd12ww3 file.  And that needs to have the INPUTDATA path used like for some other ice_in variables.  

I have a working sandbox with this fix in it and I'm about to PR, so hopefully we'll have a fix soon.  @lettie-roach, you don't need to worry about providing a PR at this point, but I will ask for you to review the PR when it comes in.  thanks!

**Comment by apcraig** [comment]
Fixed in #417
